internal/emlb/emlb.go

```diff
@@ -76,7 +76,7 @@ type EMLB struct {
 	client         *lbaas.APIClient
 	metro          string
 	projectID      string
-	tokenExchanger *TokenExchanger
+	TokenExchanger *TokenExchanger
```

Comment:
Changes from #778 and #776 being included in this PR are making this larger than it should be

Reply:
Yep, I'll be rebasing this after those are merged.

Reply:
I've removed the extra commits.